### PR TITLE
Hero accessibility

### PR DESCRIPTION
**Pull request by kfriedman**
This PR addresses #59 and #60. It:

- Removes the "category" heading
- Changes the main header to a `h1`.
- Updates the main header copy

### CODE REVIEW

File: src/app/components/Hero/Hero.jsx [hunk 1]
```diff
@@ -16,10 +16,10 @@ class Hero extends React.Component {
         <div className="Hero__container nypl-full-width-wrapper">
           <div className="Hero__text nypl-column-three-quarters nypl-column-offset-one">
             <div className="Hero__text__HeroTitle">
-              <h3>{this.props.heroData.category}</h3>
-              <p className="Hero__text__HeroTitle__des">
+              {this.props.heroData.category ? <p>{this.props.heroData.category}</p> : ''}
```
Comment (EdwinGuzman) [inline src/app/components/Hero/Hero.jsx]: I think they just want to completely remove the "category" from the hero? Maybe @courtneymcgee or @wlla can clarify. If so, we wouldn't need this property in the config.

Comment (kfriedman) [reply src/app/components/Hero/Hero.jsx]: Um, I guess I wasn't sure if this was being removed _permanently_ (i.e. for future Staff Picks also). So I thought the best thing to do for now was to make the display determined by config. But I'm okay with removing it too!

Comment (EdwinGuzman) [reply src/app/components/Hero/Hero.jsx]: I honestly have no clue what the Hero content will contain. Everything else is good so let's merge and update later once we have better requirements.

**Comment by wlla** [comment]
@courtneymcgee should make the final call/confirmation, but yeah _permanent_ removal (i.e. for future Staff Picks also) is what I had in mind. Context: When redesigned app is used more broadly, we'll include the full breadcrumb pattern which includes that category information. Plus, it's current placement is a wrench in clear heading organization.

**Pull request merged**